crates/pixi_build_frontend/src/tool/cache.rs

```diff
@@ -211,41 +342,35 @@ mod tests {
     use rattler_conda_types::{ChannelConfig, MatchSpec, NamedChannelOrUrl, ParseStrictness};
     use reqwest_middleware::ClientWithMiddleware;
 
-    use super::ToolCache;
+    // use super::ToolCache;
     use crate::{
         tool::{ToolContext, ToolSpec},
         IsolatedToolSpec,
     };
 
     #[tokio::test]
     async fn test_tool_cache() {
-        let mut cache = ToolCache::new();
+        // let mut cache = ToolCache::new();
         let mut config = Config::default();
         config.default_channels = vec![NamedChannelOrUrl::Name("conda-forge".to_string())];
 
         let auth_client = ClientWithMiddleware::default();
         let channel_config = ChannelConfig::default_with_root_dir(PathBuf::new());
 
-        let channels = config
-            .default_channels
-            .iter()
-            .cloned()
-            .map(|c| c.into_channel(&channel_config).unwrap())
-            .collect();
-
-        let tool_context = ToolContext::builder(channels)
+        let tool_context = ToolContext::builder()
             .with_client(auth_client.clone())
             .build();
 
-        cache.context = tool_context;
+        // cache.context = tool_context;
 
         let tool_spec = IsolatedToolSpec {
             specs: vec![MatchSpec::from_str("cowpy", ParseStrictness::Strict).unwrap()],
             command: "cowpy".into(),
+            channels: Vec::from([NamedChannelOrUrl::Name("conda-forge".to_string())]),
```

Comment:
I believe `"conda-forge".parse().unwrap()` would also work just FYI. Although this one is also good since there is no unwrap.